game/src/info/lane.rs

```diff
@@ -89,7 +89,7 @@ pub fn debug(ctx: &EventCtx, app: &App, details: &mut Details, id: LaneID) -> Ve
         ));
     }
 
-    if let Some(types) = l.get_turn_restrictions(r) {
+    if let Some(types) = l.get_turn_restrictions(r).map(|types| -> std::collections::BTreeSet<_> {types.collect()}) { // TODO is BTreeSet really needed for debugging? Would Vec be enough?
```

Comment:
Deterministic order is useful. Otherwise if you idle on the lane debug tab with the sim running, it recalculates this as time passes. If the order changes, it'll be quite jittery. But `get_turn_restrictions` has a fixed order, so `types.collect::<Vec<_>>()` would be fine